src/codegen.cpp

```diff
@@ -7010,9 +7008,6 @@ extern "C" void *jl_init_llvm(void)
 #else
         .setOptLevel(jl_options.opt_level == 0 ? CodeGenOpt::None : CodeGenOpt::Aggressive)
 #endif
-#ifdef USE_ORCMCJIT
-        .setUseOrcMCJITReplacement(true)
```

Comment:
Because it never was? https://github.com/JuliaLang/julia/commit/a38fa5fbab3f1ec2cc4a7a4b6e968767da41a153#diff-1bddf7f4e1c1c6e8fb26d87d1ded1460R15